spec/locales-currencies-tz.html

```diff
@@ -155,41 +164,44 @@ <h1>IsWellFormedCurrencyCode ( _currency_ )</h1>
 
       <p>
-        The IsValidTimeZoneName abstract operation verifies that the _timeZone_ argument (which must be a String value) represents a valid Zone or Link name of the IANA Time Zone Database.
-      </p>
-      <p>
-        The abstract operation returns true if _timeZone_, converted to upper case as described in <emu-xref href="#sec-case-sensitivity-and-case-mapping"></emu-xref>, is equal to one of the Zone or Link names of the IANA Time Zone Database, converted to upper case as described in <emu-xref href="#sec-case-sensitivity-and-case-mapping"></emu-xref>. It returns false otherwise.
+        The abstract operation IsValidTimeZoneName takes argument _timeZone_, a String value, and verifies that it represents a valid Zone or Link name of the IANA Time Zone Database.
       </p>
+
+      <emu-alg>
+        1. If one of the Zone or Link names of the IANA Time Zone Database is an ASCII-case-insensitive match of _timeZone_ as described in <emu-xref href="#sec-case-sensitivity-and-case-mapping"></emu-xref>, return *true*.
+        1. If _timeZone_ is an ASCII-case-insensitive match of *"UTC"*, return *true*.
+        1. Return *false*.
+      </emu-alg>
     </emu-clause>
 
+    <emu-note>
+      Any value returned from DefaultTimeZone must be recognized as valid.
+    </emu-note>
+
     <emu-clause id="sec-canonicalizetimezonename" aoid="CanonicalizeTimeZoneName">
       <h1>CanonicalizeTimeZoneName</h1>
 
       <p>
-        The CanonicalizeTimeZoneName abstract operation returns the canonical and case-regularized form of the _timeZone_ argument (which must be a String value that is a valid time zone name as verified by the IsValidTimeZoneName abstract operation). The following steps are taken:
+        The abstract operation CanonicalizeTimeZoneName takes argument _timeZone_, a String value that is a valid time zone name as verified by IsValidTimeZoneName, and returns the canonical and case-regularized form of _timeZone_.
       </p>
 
       <emu-alg>
-        1. Let _ianaTimeZone_ be the Zone or Link name of the IANA Time Zone Database such that _timeZone_, converted to upper case as described in <emu-xref href="#sec-case-sensitivity-and-case-mapping"></emu-xref>, is equal to _ianaTimeZone_, converted to upper case as described in <emu-xref href="#sec-case-sensitivity-and-case-mapping"></emu-xref>.
-        1. If _ianaTimeZone_ is a Link name, let _ianaTimeZone_ be the corresponding Zone name as specified in the *"backward"* file of the IANA Time Zone Database.
+        1. Let _ianaTimeZone_ be the String value of the Zone or Link name of the IANA Time Zone Database that is an ASCII-case-insensitive match of _timeZone_ as described in <emu-xref href="#sec-case-sensitivity-and-case-mapping"></emu-xref>.
+        1. If _ianaTimeZone_ is a Link name, let _ianaTimeZone_ be the String value of the corresponding Zone name as specified in the file <code>backward</code> of the IANA Time Zone Database.
```

Comment:
are the `<code>` tags necessary vs `*`?

Reply:
`*backward*` suggests that "backward" is an ECMAScript Language Type Value and renders as `<emu-val>backward</emu-val>`, which isn't right. ``` `backward` ``` suggests that it is Source Code, which also isn't right (even though it would render as `<code>backward</code>`, which _looks_ fine). Ultimately this isn't a case covered well by [ecmarkup](https://tc39.es/ecmarkup/#emd-overview) AFAICT, but I was following the lead of https://tc39.es/ecma262/#sec-shared-memory-guidelines ("_… such as <code>LOCK</code>-prefixed instructions on x86_"). However, `~backward~` would suggest that it is an instance of a specification type ("_specification artefacts that do not necessarily correspond to any specific entity within an ECMAScript implementation_" per [ECMA-262](https://tc39.es/ecma262/#sec-ecmascript-specification-types)) and would render as `<emu-const>backward</emu-const>`, which I think can work. What do you think?

Reply:
I agree that this is the best representation in that case, but `~` might work also. One question would be: do we have any other instances of either? Should we align?

Reply:
Yes, there's a reference to Unicode Character Database file SpecialCasings.txt at [String.prototype.toLocaleLowerCase](https://tc39.es/ecma402/#sup-string.prototype.tolocalelowercase) and files UnicodeData.txt and SpecialCasings.txt at [ECMA-262 String.prototype.toLowerCase](https://tc39.es/ecma262/#sec-string.prototype.tolowercase) and file CaseFolding.txt at [ECMA-262 Canonicalize](https://tc39.es/ecma262/#sec-runtime-semantics-canonicalize-ch), which I was planning to address after this. In fact, let me open an ECMA-262 PR now.

Reply:
Update: The ECMA-262 editors expressed a preference for `<code>…</code>` as used in this PR.